models/staging/jaffle_shop/src_jaffle_shop.yml

```diff
@@ -0,0 +1,28 @@
+version: 2
+
+# Extra Credit:
+#Configure your Stripe payments data to check for source freshness.
+#Run snapshot-freshness.
+#Check the exemplar on the next page for the open ended exercises

```

Comment:
Feel free to remove any extra code/descriptions like this! It's a good habit to build to ensure your code is clean. Also if you wanted to, feel free to include source freshness tests :) I know Jose implemented them on the actual BIR project.